src/store/actions/Chat.ts

```diff
@@ -1,10 +1,15 @@
 import { createAsyncThunk } from "@reduxjs/toolkit";
```

Comment:
Consider following the Redux style guide/best practices referenced in https://github.com/sparkletown/internal-sparkle-issues/issues/244

Of particular note, which reduces a lot of Redux boilerplate, and is considered best practice:

- https://redux-toolkit.js.org/